### PR TITLE
Add state info

### PR DESCRIPTION
**Pull request by marksou**
First draft

### CODE REVIEW

**Comment by marksou** [comment]
Fixes #1440

File: tyk-docs/content/getting-started/tutorials/create-security-policy.md [hunk 1]
```diff
@@ -56,12 +56,28 @@ Adding a policy to a Community Edition Tyk Gateway is very easy. Polices are loa
...
+*   `{API-NAME}`: The name of the API that is being granted access to (this is not required, but helps when debugging or auditing).
+*   `POLICY NAME`: The name of this security policy.
+
+The important elements:
+
+*   `access_rights`: A list of objects representing which APIs that you have configured to grant access to.
+*   `rate` and `per`: The number of requests to allow per period.
+*   `quota_max`: The maximum number of allowed requests over a quota period.
+*   `quota_renewal_rate`: how often the quota resets, in seconds. In this case we have set it to renew every hour.
+*   `state`: New from **v3.0**, this can be used instead of `active` and `is_inactive`. You can use the following values:
```
Comment (devsergiy) [inline tyk-docs/content/getting-started/tutorials/create-security-policy.md:77]: Should we mention that if we pass `state` any values of `active` and `is_inactive` will be overwritten?

Comment (marksou) [reply tyk-docs/content/getting-started/tutorials/create-security-policy.md:77]: @spetrunin Good point, will add.

**Comment by marksou** [comment]
@spetrunin Can you review again?

**Review by devsergiy — APPROVED** [review]

**Pull request merged**